docs/src/layouts/MainLayout.vue

```diff
@@ -183,27 +161,47 @@ export default defineComponent({
       ...Menu
     ]
 
-    const socialLinks = [
-      { icon: fabGithub, href: 'https://github.com/quasarframework' },
-      { icon: 'message', href: 'https://discord.com/invite/5TDhbDg' },
-      { icon: 'forum', href: 'https://github.com/quasarframework/quasar/discussions/' },
-      { icon: fabTwitter, href: 'https://twitter.com/quasarframework' },
-      { icon: fabFacebookSquare, href: 'https://www.facebook.com/QuasarFramework' }
+    const navItems = [
+      {
+        label: 'Docs',
+        path: 'docs'
+      },
+
+      {
+        label: 'Components',
+        path: 'components'
+      },
+      {
+        label: 'Become sponsor',
+        path: 'sponsors-and-backers'
+      },
+      {
+        label: 'Team',
```

Comment:
Check out https://app.zeplin.io/project/5f3fa428cd277137b15867a6/screen/60b8f0e3e82cb3a3de85ff88
I think the grouping you need to do is there, those mockups should be up-to-date

Reply:
If I got Nick correctly, he said for the this release we won't be having anything under services. That's why I left those out. I have left Nick a message in zeplin to confirm this.